### ISSUE TITLE
Less narrow pages when TOC is disabled

### ISSUE THREAD

**Issue by helins** [Convex-Dev/convex-web#183]
Overall pages are very narrow. Could we make them larger at least when TOC is disabled?

Otherwise actual code examples are hard to follow because of X scrolling:

![Screenshot from 2021-09-09 10-20-04](https://user-images.githubusercontent.com/17478152/132650129-f9fd59c2-a9e1-4df2-bfc5-24ea82292864.png)

**Comment by pedrorgirardi** [comment]
Good idea!

**Comment by helins** [comment]
I think there is a slight problem with the width of TOC. First picture is fine, second one is all squashed.

Part of the solution is probably TOC being completely separate from the page itself with a min-width, and disappearing when screen width becomes too small.

The most important is that the page shouldn't be too narrow, even with TOC. TOC is more of a bonus, available only if screen width allow. If there is not enough room for it, it's best to simply hide it.

![Screenshot from 2021-09-09 14-06-21](https://user-images.githubusercontent.com/17478152/132683135-997de9f3-47a7-4135-ad62-c5dee313b068.png)
![Screenshot from 2021-09-09 14-06-26](https://user-images.githubusercontent.com/17478152/132683137-4ae5ded0-a817-4504-a5b5-9e5fed6f0654.png)
